src/EncryptService.php

```diff
@@ -46,10 +46,10 @@ function encrypt($text) {
     $key = \Drupal::getContainer()->get('key_manager')->getKey($settings->get('encryption_key'));
 
     // Load the encryption method.
-    $enc_method = \Drupal::getContainer()->get('encryption')->createInstance($settings->get('encryption_method'));
+    $enc_method = \Drupal::getContainer()->get('plugin.manager.encrypt.encryption_methods')->createInstance($settings->get('encryption_method'));
 
     // Return the encrypted string.
-    return $enc_method->encrypt($text, $key->getContents());
+    return $enc_method->encrypt($text, $key->getKeySettings());
```

Comment:
This is a bit surprising. This should be returning a Key entity. This function is available from the plugin manager. We need to take a look at this and replace it with \Drupal::entityManager('key')->get($key) or something similar...